int2lm/Dockerfile

```diff
@@ -0,0 +1,44 @@
+# Build stage with Spack pre-installed and ready to be used
+FROM c2sm/nvhpc:21.3-devel-cuda_multi-ubuntu20.04-mpich
+ENV INT2LM_SPEC="int2lm@c2sm-master%nvhpc ^cmake%gcc"
+
+# install dummy slurm
+RUN apt-get -yqq update \
+ && apt-get -yqq install --no-install-recommends \
+        slurm \
+ && locale-gen en_US.UTF-8 \
+ && pip3 install boto3 \
+ && rm -rf /var/lib/apt/lists/*
+
+RUN echo "  slurm:" >> /root/.spack/packages.yaml && \
+    echo "      buildable: false" >> /root/.spack/packages.yaml && \
+    echo "      externals:" >> /root/.spack/packages.yaml && \
+    echo "      - spec: slurm%gcc" >> /root/.spack/packages.yaml && \
+    echo "        prefix: /usr" >> /root/.spack/packages.yaml
+
+# install libtree
+RUN git clone https://github.com/haampie/libtree.git /opt/libtree && \ 
+    cd /opt/libtree && \
+    make LDFLAGS=-static
+
+RUN ln -fs /opt/libtree/libtree /usr/bin/libtree 
+
+# install int2lm
+RUN --mount=type=ssh spack install --only dependencies $INT2LM_SPEC
+
+RUN --mount=type=ssh spack install -v --only package $INT2LM_SPEC
+
+RUN echo $(spack find --paths int2lm | grep -o -P '/opt/spack-install/.*')/bin/int2lm > /opt/int2lm
```

Comment:
Does this work?
Spack installs no int2lm-exe, rather the exes name is directly bin